exporter/exportertest/sink_exporter_test.go

```diff
@@ -30,18 +32,14 @@ func TestSinkTraceExporter(t *testing.T) {
 	}
 	want := make([]consumerdata.TraceData, 0, 7)
 	for i := 0; i < 7; i++ {
-		if err := sink.ConsumeTraceData(context.Background(), td); err != nil {
-			t.Fatalf("Wanted nil got error")
-		}
+		err := sink.ConsumeTraceData(context.Background(), td)
+		require.Nil(t, err)
 		want = append(want, td)
 	}
 	got := sink.AllTraces()
-	if !reflect.DeepEqual(got, want) {
-		t.Errorf("Mismatches responses\nGot:\n\t%v\nWant:\n\t%v\n", got, want)
-	}
-	if sink.Name() != "sink_trace" {
-		t.Errorf("Wanted sink_trace got %s", sink.Name())
-	}
+	assert.True(t, reflect.DeepEqual(got, want))
```

Comment:
You can use `assert.Equal` directly, e.g.: `assert.Equal(t, want, got)` it will give a better message in case of error.